src/utils/growthbook-utils.ts

```diff
@@ -45,3 +45,18 @@ export const isShowStagingBuildStatusWhitelistedRepo = (
     false
   )
 }
+
+export const isCloudmersiveEnabled = (
+  growthbook: GrowthBook<FeatureFlags> | undefined
+): cmConfigType => {
+  const defaultConfig = {
+    is_enabled: false,
```

Comment:
nit: should we default to true instead to be safe if growthbook is down?

Reply:
@dcshzj wah bro that was my initial approach. but i was worried that eng in the future might toggle gb to be OFF -> but then CM is still enabled cuz if the feature toggle off -> CM still on cuz null defaults to ON 
I ok to j leave a comment in gb about this unintuitive behaviour, i do have a preference for it being dafult on. Thoughts

Reply:
Ah I see okay yeah that makes sense, I think probably won't need a comment here but maybe can put inside growthbook instead?